### ISSUE TITLE
Add margin option to auto resizer

### ISSUE THREAD

**Issue by AndrewLeedham** [contentful/ui-extensions-sdk#1542]
It would be useful to provide margin to be added to the size calculated by the auto resizer in Contentful Apps. For example using the forma-36 Autocomplete component with the auto resizer the shadow is cut-off:
![image](https://github.com/contentful/ui-extensions-sdk/assets/5557458/22a327de-e1e7-4578-8920-529aa4a13467)
Adding 5-10px of margin here would solve the issue.

When using `@contentful/react-apps-toolkit` it would look something like this:
```
function Component() {
  useAutoResizer({ absoluteElements: true, margin: 10 });
}
```

**Comment by mgoudy91** [comment]
Just to make sure, you've tried using something like
`sdk.window.startAutoResizer({ absoluteElements: true });` ?

It sounds like you have, but I thought I'd drop the snippet as it's a not well-documented feature of the sdk.

**Comment by AndrewLeedham** [comment]
@mgoudy91 Yes I am actively using `absoluteElements: true` as your `<Autocomplete/>` component positions the dropdown absolutely.

**Comment by mgoudy91** [comment]
> @mgoudy91 Yes I am actively using `absoluteElements: true` as your `<Autocomplete/>` component positions the dropdown absolutely.

Sounds good, I'll have this thread updated when the long term fix is in place
